src/backend/web/validation.js

```diff
@@ -73,6 +73,18 @@ const queryValidationRules = [
     .isIn(['post', 'author'])
     .withMessage('invalid filter value')
     .bail(),
+
+  check('perPage')
```

Comment:
I don't see the validation tests for these.  Add test cases for:

- perPage is empty `&perPage=`
- perPage is some garbage string `&perPage=invalid`
- perPage is a number smaller than 1-10 `&perPage=0`
- perPage is a number larger than 1-10 `&perPage=11`
- perPage is valid `&perPage=3`

same idea for `page`

Reply:
@humphd sorry bit of an express validation noob here, I though the `check('perPage')` and `check('page')` validation chains do that? most of the test cases give out the correct responses, page or perPage out of range gives 400, querying anything but an int will give a 400 as well. Only the first one doesn't seem to work

Reply:
This is for our web route tests, to make sure that our web server rejects any params that aren't legal.  See https://github.com/Seneca-CDOT/telescope/blob/master/test/query.test.js which is where you need to add tests, since you're changing how our web routes work, you need new tests so we don't break it.

Reply:
I've coded the validation tests but just one question. What should the response be for empty params e.g. `&perPage=` and `&page=` should it be valid 200 or invalid 400. Right now the query accepts the value as '' (empty string?) when passed like that so i need to code validation logic for that as well.

Reply:
I think passing `&perPage=` and `&page=` is probably a user bug (i.e., 400) and we should error.  If you set these params in the URL, we'll expect you to give proper values.  If you leave them off, we'll do sane defaults.

Reply:
Ok, I'm trying to get it to reject `&perPage=` and `&page=` since it express validation still accepts those values right now, will push a commit soon